<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pulse</class>
 <widget class="QWidget" name="Pulse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pls_take">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>194</y>
     <width>106</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TAKE</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="pulse_system">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>284</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>SYSTEM</string>
   </property>
   <widget class="QPushButton" name="pvwID">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable preview ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoTakeOn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto take ON</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="autoTakeOFF">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto take OFF</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="pulse_out">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>189</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>OUTPUTS</string>
   </property>
   <widget class="QPushButton" name="pls_bgFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bg
FRAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_bgLive">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bg
LIVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_pip">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PIP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_logo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGO</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="pulse_in">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>24</y>
     <width>481</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>INPUTS</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pls_in_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>VGA
PLAT</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_in_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA
TRAD</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_in_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA 3</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_in_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA 4</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_in_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA 5</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_in_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>VGA 6</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_dvi1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>DXP
OUT2</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_dvi2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC
PRO</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_sdi1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SDI
PALT</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_sdi2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SDI 2</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_black">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>BLACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_freeze">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>FREEZE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pls_unfreeze">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>UNFREEZE</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>228</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="pls_status_icon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressources.qrc">:/icons/img/ko.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblPulse">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ANALOGWAY PULSE PLS300</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>2</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>-2</y>
      <width>122</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="pls_on_off">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="pls_on">
       <property name="text">
        <string>ON</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pls_off">
       <property name="text">
        <string>OFF</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>489</width>
     <height>352</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 228, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pls_take</zorder>
  <zorder>pulse_system</zorder>
  <zorder>pulse_out</zorder>
  <zorder>pulse_in</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
